packages/trace-viewer/src/ui/uiModeView.tsx

```diff
@@ -371,7 +373,7 @@ export const UIModeView: React.FC<{}> = ({
       <div className='title'>UI Mode disconnected</div>
       <div><a href='#' onClick={() => window.location.href = '/'}>Reload the page</a> to reconnect</div>
     </div>}
-    <SplitView sidebarSize={250} minSidebarSize={150} orientation='horizontal' sidebarIsFirst={true} settingName='testListSidebar'>
+    <SplitView sidebarSize={isUnderTest ? 350 : 250} minSidebarSize={150} orientation='horizontal' sidebarIsFirst={true} settingName='testListSidebar'>
```

Comment:
The problem is that our run buttons are overlaying the tags so the element is not clickable after moving the cursor over it.

https://github.com/microsoft/playwright/assets/17984549/1e937371-9585-47fa-9c3c-2953c6b4fa9b